### DISCUSSION TITLE
How to fix the bank app?

### DISCUSSION THREAD

**Discussion by Onewpus** [tribalfs/GalaxyMaxHzPub#243] [Q&A]
Is there any way to fix the Galaxy Max Hz app to enter bank apps?

**Comment by tribalfs** [comment]
Try to check the latest beta version with the option to temporarily turn off GMH.

https://github.com/tribalfs/GalaxyMaxHzPub/discussions/147

**Comment by Onewpus** [comment]
When will the official edition of 90 be released?
